### PR TITLE
adds an option for handling bottom side PCB file

### LINKED ISSUE #8: Option for bottom layer
Hey there! Awesome work here.

Is there any possibility to do it with the bottom layer also?

Regards.

### PR DESCRIPTION
**Pull request by joysfera**
Adds "-b" option for handling bottom side solder paste files. Fixes #8

### CODE REVIEW

File: gerber_to_scad.py [hunk 1]
```diff
@@ -400,14 +400,15 @@ def process(outline_file, solderpaste_file, stencil_thickness=0.2, include_ledge
         ledge_polygon = ledge_polygon - polygon(cutter)
 
         ledge = utils.down(
-            ledge_height - stencil_thickness
+            0 if bottom_side else (ledge_height - stencil_thickness)
```
Comment (Aeroeon) [inline gerber_to_scad.py:403]: Accidental 0 here?

Comment (joysfera) [reply gerber_to_scad.py:403]: nope

Comment (joysfera) [reply gerber_to_scad.py:403]: > Accidental 0 here?

nope

**Comment by kirberich** [comment]
Hey, really sorry for not getting back to you about this for several years - I had completely forgotten about it, and then apparently ended up implementing this differently, even though I think your solution is nicer to read.

So this works now, by passing the `flip_stencil` option - but if you'd like to rebase this PR and replace the "flip_stencil" flag with your implementation I'd happy swap them out

**Pull request closed without merging**